src/components/navbar.js

```diff
@@ -48,7 +50,7 @@ const Navbar = ({ showBanner }) => {
   }, [])
 
   const [promotionOpen, setPromotionOpen] = useState(showBanner)
-  const closePromotion = useCallback(() => setPromotionOpen(false), [])
+  const closePromotion = useCallback(() => setPromotionOpen(false))
```

Comment:
React Hook useCallback does nothing when called with only one argument.
The `[]` dependency list assures that the callback doesn't change.